pom.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) ${project.inceptionYear} DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>nl.knaw.dans.shared</groupId>
+        <artifactId>dans-java-prototype</artifactId>
+        <version>1.32</version>
+    </parent>
+    <groupId>nl.knaw.dans.easy</groupId>
+    <artifactId>ddm</artifactId>
+    <version>2.x-SNAPSHOT</version>
+    <name>Dans Dataset Metadata Library (DDM)</name>
+    <inceptionYear>2014</inceptionYear>
```

Comment:
Change to 2014-17 for consistency with other projects. They run `mvn license:format` again to update the license headers.